docs/intro-react.md

```diff
@@ -0,0 +1,476 @@
+      <Text>I am a also cat!</Text>
+    </View>
+  );
+}
+
+export default function Cafe() {
+  return (
+    <View>
+      <Cat />
+      <Cat />
+      <Cat />
+    </View>
+  );
+}
+```
+
+You can put as many cats in your cafe as you like. Each `<Cat>` renders a unique element that can have its own data—which you pass via props.
+
+## Props
+
+**Props** is short for “properties.” Props let you customize React components. For example, here each `<Cat>` passes a different `name` for the `Cat` function to render:
+
+```SnackPlayer name=Multiple%20Props
+import React from 'react';
+import { Text, View } from 'react-native';
+
+function Cat(props) {
+  return (
+    <View>
+      <Text>Hello, I am {props.name}!</Text>
+    </View>
+  );
+}
+
+export default function Cafe() {
+  return (
+    <View>
+      <Cat name="Maru" />
+      <Cat name="Jellylorum" />
+      <Cat name="Spot" />
+    </View>
+  );
+}
+```
+
+Most of React Native’s Core Components can be customized with props, too. For example, when using [`Image`](image), you pass it a `prop` named [`source`](image#source) to define what image it shows:
+
+```SnackPlayer name=Props
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+
+export default function CatApp() {
+  return (
+    <View>
+      <Image
+        source='https://placekitten.com/g/193/110'
+        style={{
+          width: 193,
+          height: 110
+        }}/>
+      <Text>Hello, I am your cat!</Text>
+    </View>
+  );
+}
+```
+
+`Image` has [many different props](image#props), including [`style`](image#style), which accepts a JS object of design and layout related property-value pairs.
+
+> Notice the double curly braces `{{ }}` surrounding `style`‘s width and height. In JSX, JavaScript values are referenced with `{}`. This is handy if you are passing something other than a string as props, like an array or number (`<Cat food={["fish", "kibble"]} /> age={2}`). However, JS object are **_also_** denoted with curly braces: `{width: 193, height: 110}`. Therefore, to pass a JS object in JSX, you must wrap the object in **another pair** of curly braces: `{{width: 193, height: 110}}`
+
+You can add custom cat pictures to your component by:
+
+- adding an `Image` component to your `Cat` component
+- setting `Image`‘s `source` via `Cat`‘s `props` with `{props.picURI}`
+- then passing `picURI` values in each `Cat`
+
+```SnackPlayer name=Custom%20Pictures
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+
+function Cat(props) {
+  return (
+    <View>
+      <Image source={props.picURI} style={{width: 200, height: 200}}/>
+      <Text>Hello, I am {props.name}!</Text>
+    </View>
+  );
+}
+
+export default function Cafe() {
+  return (
+    <View>
+      <Cat name="Maru" picURI="https://placekitten.com/g/200/200" />
+      <Cat name="Jellylorum" picURI="https://placekitten.com/g/200/200" />
+      <Cat name="Spot" picURI="https://placekitten.com/g/200/200" />
```

Comment:
Kind of negates the point of this example to pass the same URIs. Can we come up with something?

Reply:
Well.... I wanted to make custom art for each of these... but first I wanted to make sure that all the proofreaders liked the samples ;)

So... can I draw some cats?

Reply:
I drew some cats.